framework/src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -498,16 +503,7 @@ public static void printAccount(Account reply, HttpServletResponse response, Boo
   public static byte[] getAddress(HttpServletRequest request) throws Exception {
     byte[] address = null;
     String addressParam = "address";
-    String addressStr = request.getParameter(addressParam);
-    if (StringUtils.isBlank(addressStr)) {
-      String input = request.getReader().lines()
-          .collect(Collectors.joining(System.lineSeparator()));
-      Util.checkBodySize(input);
-      JSONObject jsonObject = JSON.parseObject(input);
-      if (jsonObject != null) {
-        addressStr = jsonObject.getString(addressParam);
-      }
-    }
+    String addressStr = checkGetParam(request, addressParam);
```

Comment:
`getParam` or `checkAndGetParam` ?

Reply:
Use `checkGetParam` as the method name, because there are some checking judgments